src/python/pants/backend/python/subsystems/twine.py

```diff
@@ -72,6 +70,27 @@ class TwineSubsystem(PythonToolBase):
         ),
     )
 
+    ca_certs_path = StrOption(
+        advanced=True,
+        default="<inherit>",
```

Comment:
Should this option be deprecated? Or did you encounter a case where it might make sense to override it only for `twine`...?

Reply:
I'm waiting for @kaos to reply. I'm not sure why it exists

Reply:
From Andreas in Slack:

> According to https://github.com/pantsbuild/pants/pull/13593 my reasoning was that you’d likely don’t need to use the twine option which defaults to the global one, but I added it in case the two cert bundles would differ.  Maybe overkill/premature feature..?

Definitely using `[GLOBAL].ca_certs_path` by default was a good improvement. I lack enough real world experience to know if allowing you to use different certs than the global option is overkill or not. Thoughts @thejcannon @stuhood @benjyw @tdyas ?

Reply:
Follow up from Andreas:

> The thing to perhaps support is if you want to publish your python dist with twine to a cheeseshop pypi server with a self signed cert, you’ll need a custom cert bundle with your issuing CA in it, but you may not want to use that same bundle for all purposes as it would add a maintenance burden to keep it up to date with the world.

So sounds like this is fine.